app/ui/browser/views/navbar/location.jsx

```diff
@@ -367,11 +391,13 @@ Location.propTypes = {
   ipcRenderer: PropTypes.object.isRequired,
   navigateTo: PropTypes.func.isRequired,
   userTypedLocation: PropTypes.string.isRequired,
+  showCompletions: PropTypes.bool.isRequired,
 };
 
 function mapStateToProps(state, ownProps) {
   return {
     userTypedLocation: getUserTypedLocation(state, ownProps.page.id),
+    showCompletions: state.uiState.showCompletions,
```

Comment:
We've been using selectors (like getUserTypedLocation above). This makes it easier to restructure the state shape if we need to.